core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -60,8 +60,10 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
              translate="I18N_DASHBOARD_MY_EXPLORATIONS">
           </a>
         </li>
-        {% if can_create_collections %}
+        {% if not can_create_collections %}
```

Comment:
This has been added as a layout fix when My Collections tab isn't visible. Else we'll have to use `float: right` to keep things in position!